app/components/sy-timepicker/component.js

```diff
@@ -103,7 +104,11 @@ export default Component.extend({
    * @private
    */
   _addMinutes(value) {
-    this._change(this._add(0, value))
+    let newValue = this._add(0, value)
+
+    if (this._validate(newValue)) {
```

Comment:
I'd call it `_checkValidity` since `isValid` sounds like a property to me which it clearly isn't. I'll change this if you agree?

Reply:
`checkValidity` also sounds like the function is "doing" something complicated. Since your function is pure, it behaves like a computed property, so i'd be fine with calling it `isValid`.
I understand your point too though - just leave it as it is if you don't like `isValid` ;)

Reply:
Valid point :wink: Done!